### PR TITLE
Feature/TT-4851-organize-shared

### PR DESCRIPTION
**Pull request by gtryus**

### CODE REVIEW

File: src/components/Workflow/PlanAudioActions.tsx [hunk 1]
```diff
@@ -61,7 +61,7 @@ export function PlanAudioActions(props: IProps) {
         <StyledIconButton
           id="passageShare"
           shared={mediaShared === IMediaShare.OldVersionOnly}
-          title={t.versions}
+          title={mediaShared ? 't.resourceEdit' : t.versions}
```
Comment (sarahentzel) [inline src/components/Workflow/PlanAudioActions.tsx:64]: should this be in quotes?

File: src/crud/useShaRefUpdate.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  const [memory] = useGlobal('memory');
+  const [user] = useGlobal('user');
+
+  return (sharedResourceReferences: SharedResourceReference[]) => {
+    const t = new TransformBuilder();
+    let ops: Operation[] = [];
+    for (const sharedResourceReference of sharedResourceReferences) {
+      ops = ops.concat([
+        ...UpdateRecord(t, sharedResourceReference, user),
+        ...ReplaceRelatedRecord(
```
Comment (sarahentzel) [inline src/crud/useShaRefUpdate.ts:16]: Will you really ever be changing the shared resource?

Comment (gtryus) [reply src/crud/useShaRefUpdate.ts]: I removed that. I can't see ever changing it.

File: src/components/Workflow/PlanAudioActions.tsx (hunk 1 above)
Comment (gtryus) [reply src/components/Workflow/PlanAudioActions.tsx]: I added this string and updated crowd in and the files based on it.

**Pull request merged**